docs/cli/install-runpodctl.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Installing runpodctl
+description: "Install the command line interface for RunPod and run your first command."
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+runpodctl is an [open-source command-line interface (CLI)](https://github.com/runpod/runpodctl). You can use runpodctl to work with Pods and RunPod projects. 
+
+When you create a Pod, it comes with runpodctl installed and configured with a Pod-scoped API key. You can also run runpodctl locally.
+
+To install runpodctl on your local machine, run the appropriate command for your operating system. 
+
+<Tabs>
+  <TabItem value="mac" label="macOS" default>
+
+    ```bash
+    wget -qO- cli.runpod.net | sudo bash
+    ```
+
+  </TabItem>
+  <TabItem value="linux" label="Linux">
+
+    ```bash
+    wget -qO- cli.runpod.net | sudo bash
+    ```
+
+  </TabItem>
+  <TabItem value="windows" label="Windows">
+
+    ```bash
+    wget https://github.com/runpod/runpodctl/releases/download/v1.10.0/runpodctl-win-amd -O runpodctl.exe
```

Comment:
this is the old name of the github resource, this line should now be 
`wget https://github.com/runpod/runpodctl/releases/download/v1.12.3/runpodctl-windows-amd64.exe -O runpodctl.exe`

Reply:
Fixed